test/middleware/PostQuery.test.ts

```diff
@@ -28,25 +30,24 @@ describe('Test API key header verification', () => {
             complexity: 3,
             timestamp: 100,
             tokens: -1,
+            success: true,
+            loggedOn: 0,
         };
-        postQuery = new PostQuery(mockURI, mockProjectID, mockQueryData);
-        const newPost = () => postQuery.post();
+        const newPost = () => postQuery(mockURI, mockProjectID, mockQueryData);
 
         await expect(newPost).rejects.toThrow(`[gatelog] Query data cannot be negative.`);
```

Comment:
On my machine this test is failing because for the following reason
```
Received promise resolved instead of rejected
    Resolved to value: [SyntaxError: [gatelog] Query data cannot be negative]
```
It's resolving to the right error but not rejecting.

Reply:
fixed